db/migrate/20220810084526_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, index: true
```

Comment:
- [ ] The piece of code that wrote will create a`user_id` but in the diagram, it should be `author_id` in order to achieve that try the following:
```RUBY
t.references :author, foreign_key: { to_table: 'users' }
```